prime-angular-frontend/src/app/modules/adjudication/shared/components/review-status-content/review-status-content.component.ts

```diff
@@ -123,10 +123,10 @@ export class ReviewStatusContentComponent implements OnInit, OnChanges {
         }
 
         if (esr.statusReasonCode === EnrolmentStatusReasonEnum.IDENTITY_PROVIDER) {
-          return reasons.concat(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statuses'), esr.reasonNote, this.enrollee.identificationDocuments));
+          return reasons.concat(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statusReasons'), esr.reasonNote, this.enrollee.identificationDocuments));
         }
 
-        reasons.push(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statuses'), esr.reasonNote));
+        reasons.push(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statusReasons'), esr.reasonNote));
         return reasons;
       }, []);
   }
```

Comment:
Our status reasons in the FE are not complete, should we add the missing ones? I added them in my PR but since this is touching this area I thought maybe the right PR for it

Reply:
Sounds good they've been updated